constraints/BUCK

```diff
@@ -0,0 +1,97 @@
+constraint_setting(
+    name = "cpu",
+)
+
+constraint_value(
+    name = "aarch64",
+    constraint_setting = ":cpu",
+    visibility = ["PUBLIC"],
+)
+
+constraint_value(
+    name = "arm",
+    constraint_setting = ":cpu",
+    visibility = ["PUBLIC"],
+)
+
+constraint_setting(
+    name = "os",
+)
```

Comment:
I thought prelude had OS too?

Reply:
it does, but have to include it here. adding some corrections here, but did need to keep linux